corehq/project_limits/rate_limiter.py

```diff
@@ -39,43 +39,59 @@ def report_usage(self, scope='', delta=1):
                 rate_counter.increment(self.feature_key + limit_scope, delta=delta)
 
     def get_window_of_first_exceeded_limit(self, scope=''):
-        for limit_scope, rates in self.iter_rates(scope):
-            for rate_counter_key, current_rate, limit in rates:
+        for _limit_scope, rates in self.iter_rates(scope):
```

Comment:
Why did the `limit_scope` variable need renaming?

Reply:
It wasn't being used so I just indicated that with the underscore